nubis/puppet/discovery.pp

```diff
@@ -1,7 +1,10 @@
 include nubis_discovery
 
-# Switch to MC port once working
 nubis::discovery::service { $project_name:
```

Comment:
It feels weird to use `$project_name` here this ends up making the service discovery to be `nubis-kubernetes.service.consul` and for the other components it ends up being `kubernetes-master.nubis-kubernetes.service.consul` and `kubernetes-node.nubis-kubernetes.service.consul` to me that just sounds weird